lib/Flux.js

```diff
@@ -42,22 +51,47 @@ class ActionNotFoundError extends Error {
  * @extends Error
  */
 class StoreNotFoundError extends Error {
+  /**
+   * Contructs a new StoreNotFoundError.
+   * @constructor
+   * @param {String} path Path of the requested store
+   */
   constructor(path) {
     super(`Store not found (path=${path})`);
   }
 }
 
-function findInMatchers(needle, haystack) {
-  return _(haystack)
-    .map(([matcher, handler]) => [handler, matcher(needle)])
+/**
+ * Find the first pair of {@link Routable} and given path's query which route path matches one of the
+ * {@link Routable} matcher in the collection.
+ * @param {String} path Route path
+ * @param {Array<Array<Function|Routable>>} collection List containing pairs of matchers and their related
+ *                                          {@link Routable}s.
+ * @return {Array<Routable|Object>} The pair containing the {@link Routable} and its query if the given path matches
+ *                                  one of the {@link Routable} matcher, null otherwise.
+ */
+function findInMatchers(path, collection) {
+  return _(collection)
+    .map(([matcher, routable]) => [routable, matcher(path)])
   .find(([, query]) => query !== null);
 }
 
+/**
+ * Represent a Flux
+ * A flux is a hub for {@Store}s and {@Action}s it allow them to comunicate.
```

Comment:
"A flux is a hub for {@Store}s and {@Action}s**. I**t allow**s** them to com**m**unicate.